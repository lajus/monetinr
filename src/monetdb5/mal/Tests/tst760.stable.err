stderr of test 'tst760` in directory 'monetdb5/mal` itself:
