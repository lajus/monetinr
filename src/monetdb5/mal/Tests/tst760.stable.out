stdout of test 'tst760` in directory 'monetdb5/mal` itself:


# 08:01:30 >  
# 08:01:30 >  Mtimeout -timeout 60 Mserver "--config=/ufs/mk/monet5/Linux/etc/MonetDB5.conf" --debug=10 --set "monet_mod_path=/ufs/mk/monet5/Linux/lib/MonetDB5:/ufs/mk/opensource/MonetDB/Linux/lib/MonetDB" --set "gdk_dbfarm=/ufs/mk/monet5/Linux/var/MonetDB5/dbfarm" --set "sql_logdir=/ufs/mk/monet5/Linux/var/MonetDB5/log" --set mapi_port=42158 --set sql_port=55708 --set monet_prompt= --trace "--config=/ufs/mk/monet5/src/mal/Tests/All.conf" --dbname=mTests_src_mal  tst760.mal </dev/null
# 08:01:30 >  

# Monet Database Server V4.99.19
# Copyright (c) 1993-2004, CWI. All rights reserved.
# Compiled for i686-redhat-linux-gnu/32bit; dynamically linked.
# config:/ufs/mk/monet5/Linux/etc/MonetDB5.conf
# dbfarm:/ufs/mk/monet5/Linux/var/MonetDB5/dbfarm
# dbname:mTests_src_mal
# Visit http://monetdb.cwi.nl/ for further information.
## Polymorphic function tests
## Catching a dynamic error
#function test(b:bat[:any,:any$2],c:any$2);
#	k:= 1+c;
#	print(k);
#end test;
function user.test(b:bat[:any,:any_2],c:any_2):void;
    k := calc.+(1,c);
    io.print(k);
end test;
#
#c:=new(:int,:str);
#test(c,"hello");
#
!TypeException:user.test[1]:'calc.+' undefined in: k:any := calc.+(_4:int, c:str)

!MALException:user.test[0]:Error in cloned function
!TypeException:user.main[4]:'user.test' undefined in: _6:void := user.test(c:bat[:oid,:str], _7:str)
function user.main():void;
# Polymorphic function tests 
# Catching a dynamic error 
    c := bat.new(:oid,:str);
    user.test(c,"hello");
end main;

# 08:01:30 >  
# 08:01:30 >  Done.
# 08:01:30 >  

